src/MoBi/MoBi.csproj

```diff
@@ -207,6 +207,10 @@
       <Link>OSPSuite.PKParameters.xml</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="..\..\pre\libsbmlcs.dll">
```

Comment:
File was not included as dependencies in MoBi.exe. Since setup is done harvesting everything in the bin folder, this was not part of the setup